Rakefile

```diff
@@ -26,7 +26,7 @@ end
 desc 'Test for binstubs'
 task :check_binstubs do
   unless Dir.glob('bin/**/*.rb').empty?
-    bin_list = Gem::Specification.load('sensu-plugins-load-checks.gemspec').executables
+    bin_list = Gem::Specification.load('sensu-plugins-tomcat.gemspec').executables
```

Comment:
good old copy,pasta error on my end...now we at least have a decent skel so this should be less painful in the future.